api.rest

```diff
@@ -46,13 +49,20 @@ DELETE {{baseUrl}}/gifts/e7051758-f907-4ebf-bcaf-5b26b3198409
 
 ###
 GET {{baseUrl}}/users
+Authorization: {{authorization}}
+Origin: {{baseUrl}}
```

Comment:
`Authorization` sisältää istunnon ID-arvon, joka tarvitaan tunnistautumiseen. `Origin` taasen asettaa Origin-headeriksi vain tuon `localhost`in, joka laittaa requestit toimimaan

`middleware.tsx`-tiedoston muutokset liittyvät tähän ongelmaan. Kun lähetti `api.rest`-tiedostosta requestin, ei Origin ollutkaan `localhost` joka rikkoi tämän

Reply:
Tämä on yksi tapa. `middleware.tsx` tiedoston kommentissa on vaihtoehtoinen ratkaisu.